Automattic/jetpack
# New shell command for easier access to the database.

#### Changes proposed in this Pull Request:
Introducing the new command: `yarn docker:mysql`.
It reads the environment variables and initializes a mysql client session.
Logging into your mysql has never been easier!

Introducing the new command: `yarn docker:run-extras`.
It runs the file `docker/bin/run-extras.sh`.
The whole purpose of a separate file is to run commands that can be run during both:
1. The container initialization (run by `docker/bin/run.sh`)
2. The active container (run by `yarn docker:run-extras`).

This will let us modify the container without the need for everybody to rebuild it.

#### Jetpack product discussion
`p1596825139021600-slack-jetpack-crew`

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
1. Use the docker environment.
2. Run `yarn docker:run-extras` to run the extras
3. Run `yarn docker:compose exec wordpress cat /root/.my.cnf` to confirm that the MySQL config file was created.
3. Run `yarn docker:db`, and confirm that you actually are inside your database.
4. Run `yarn docker:compose rm -sf wordpress` to remove your docker container.
**Danger: this will remove your container with any custom changes you made to it that are not in the `docker-compose` files.** _Although you shouldn't have made any custom changed to your docker containers in the first place 😉._
5. Run `yarn docker:up -d` to restart the container. The `jetpack_wordpress` container is supposed to be rebuilt.
6. Run `yarn docker:db` to confirm that you can once again access the MySQL client.

#### Proposed changelog entry for your changes:
New shell command `yarn docker:db` for easier access to the database
